internal/admin/server.go

```
@@ -3,58 +3,68 @@
-		ca:         certAuth,
-		adminUsers: make(models.AdminUsers, 0),
+func New(cfg *models.Config, certAuth *ca.CertificateAuthority) (*Server, error) {
+	mongoClient, err := db.NewMongoClient(cfg.MongoURI, cfg.MongoDB)
+	if err != nil {
+		return nil, fmt.Errorf("failed to connect to MongoDB: %w", err)
 	}
+
+	appRepo := repository.NewMongoAppRepository(mongoClient, cfg.MongoAppsColl)
+	userRepo := repository.NewMongoUserRepository(mongoClient, cfg.MongoUsersColl)
+
+	db.EnsureIndexes(mongoClient, cfg)
```

golangci-lint (1.64.8)

Error return value of `db.EnsureIndexes` is not checked

(errcheck)